resources/views/logo.blade.php

```diff
@@ -0,0 +1,55 @@
+@extends("layout.public", ["excludeBigBubbles" => true])
+
+@php
+    $materials = [
+        [
+            'key' => 'brand_book',
+            'file_type' => 'pdf',
+            'file_size' => '2.4 MB',
+            'download_url' => url('materials/blumilk_brand_book_2024.pdf'),
```

Comment:
Is it okey for us that file type and file size is added manually?

For file type we can use:
```php
$brandBookFilename = "blumilk_brand_book_2024.pdf";

$fileType =  Str:afterLast($brandBookFilename, ".");
 ```
 Or we can use PHP builtin function [mime_content_type](https://www.php.net/manual/en/function.mime-content-type.php)
 
 Ref:
 - https://laravel.com/docs/11.x/strings#method-str-after-last
 - https://www.php.net/manual/en/function.mime-content-type.php
 
 ---
 
 For filesize we can use PHP builtin function [filesize](https://www.php.net/manual/en/function.filesize.php) and convert it to MB.
 
 or[ Storage facade](https://laravel.com/docs/11.x/filesystem#file-metadata)

Reply:
Fixed

Reply:
Will you replace `'` with `"`?

Reply:
Done :)